UnityProject/Assets/Scripts/Items/Implants/Organs/Heart.cs

```diff
@@ -156,6 +161,15 @@ public void Heartbeat(float efficiency)
 				var BloodToGive = circulatorySystem.ReadyBloodPool.Take((implant.BloodThroughput / totalWantedBlood) * pumpedReagent);
 				implant.BloodPumpedEvent(BloodToGive);
 			}
+			if (RelatedPart.HealthMaster.IsDead) return; //For some reason the heart will randomly still continue to try and beat after death.
+			if (RelatedPart.BloodContainer.CurrentReagentMix.MajorMixReagent == salt || RelatedPart.BloodContainer.AmountOfReagent(salt) > danagerSalt)
```

Comment:
this would mean that they would be more salt than blood

Reply:
yes

Reply:
well, it would have to be more than the amount of blood in the body, I think a good percentage would be )( 10 / 300 (amount of blood) so like 0.033 percentage of the bloody Salt then trigger it

Reply:
im doing it so it can act just like in SS13

Reply:
an entire bottle of a salt shaker on UnityStation currently contains 0.021 (units?) of salt, I have made it 0.020 (or 2% now) to replicate the same effect once the player drinks that much salt